fabfile.py

```diff
@@ -183,6 +183,32 @@ def update_vendor_config():
         system("cd %s ; git pull" % config_file_dir)
 
 
+@task
+def docker_piwik_database_backup():
+    """
+    To be launched with fab -c configs/bluenove-server-configs/dev_staging.rc docker_piwik_database_backup
+    """
+    # env.docker_api_version = '1.17.05'
+    env.docker_tunnel_remote_port = 22024
+    from dockerfabric import cli
+    from dockerfabric.utils.files import temp_dir
+    cwd = os.getcwd()
+    if env.docker_piwik:
+        with temp_dir() as remote_tmp:
+            print remote_tmp
+            from fabric.contrib import files
+            if files.exists("/home/assembl_user/piwik-docker-compose/docker-compose.yml"):
+                run("cd piwik-docker-compose && docker-compose up -d")
+                run("docker exec -d piwikdockercompose_db_1 mysqldump --result-file=dump.sql")
+            else:
+                run("git clone git@github.com:siutin/piwik-docker-compose.git")
```

Comment:
Again, these are just checking that docker-piwik is installed or not. That is not the responsibility of this task. You should instead check, via rc files, if the targeted server has a piwik container or a piwik folder. Then, run the `mysqldump` and move the file appropriately.